### PR TITLE
New Icon: Pico.CSS

### LINKED ISSUE #8289: Icon Request: Pico.css
### Brand Name

Pico.css

### Website

https://picocss.com/

### Popularity Metric

Popular minimalistic framwork of CSS, with raising popularity.

similarweb_pico.css

### Official Resources for Icon and Color

pico.css

### Additional Comments

- Color: The color it's self is just "black over white", other brand color is blue, is used in main theme.
- LICENSE: Te code have an MIT License, but don't know if the icon have license
- Acceptance: It's a rainsing in popularity framework, with a very comfortable experience that automates the styling of the pages.

### Contributing

Yes

### PR DESCRIPTION
**Pull request by ripzerotwo**
**Issue:** closes #8289 

**Similarweb rank:** [314,971](https://www.similarweb.com/website/picocss.com/#overview)

### Checklist
  - [X] I updated the JSON data in `_data/simple-icons.json`
  - [X] I optimized the icon with SVGO or SVGOMG
  - [X] The SVG `viewbox` is `0 0 24 24`

### Description
color grabbed from website using color pick eyedropper.
picodotcss

### CODE REVIEW

**Comment by ripzerotwo** [comment]
question if color should just be black.

**Comment by LitoMore** [comment]
Hi @lucaslarroche, may I ask for approval from you on this PR? Thank you!

File: _data/simple-icons.json [hunk 1]
```diff
@@ -8527,6 +8527,16 @@
...
         },
+        {
+            "title": "Pico.CSS",
+            "hex": "12191F",
+            "source": "https://user-images.githubusercontent.com/90356347/215237356-83b5d992-65be-464e-bccd-63ff62ec5579.svg",
+            "aliases": {
+                "aka": [
+                    "Picodotcss"
+                ]
+            }
```
Comment (adamrusted) [inline _data/simple-icons.json:8538]:
```suggestion
            "source": "https://picocss.com"
```

There's an SVG in the header of the site, and we don't need to worry about the alias.

File: icons/picodotcss.svg
Comment (adamrusted) [inline icons/picodotcss.svg:1]: Not reviewed this icon yet. Will likely be different to the source once source updated.

**Review by adamrusted — CHANGES_REQUESTED** [review]

**Comment by PeterShaggyNoble** [comment]
Note that while [the ranking](https://www.similarweb.com/website/picocss.com/) of the website has dropped since this PR was opened, putting it below our 500k threshold, it's [GH stars](https://github.com/picocss/pico) still put it within our scope. However, development seems to have stalled with [no commits to their primary repo](https://github.com/picocss/pico/graphs/commit-activity) since April.

Should we label this as https://github.com/simple-icons/simple-icons/labels/out%20of%20scope for now, pending further monitoring?

This brings to light a possible issue with us using GH stars as a metric of popularity as they are not a reflection of current, real time popularity in the same way Similarweb is, rather they reflect a cumulative popularity over the lifetime of a project. Perhaps we need to revisit _how_ we use stars as a metric of popularity but that discussion should be had separately.

**Comment by PratyushJayachandran** [comment]
> Note that while [the ranking](https://www.similarweb.com/website/picocss.com/) of the website has dropped since this PR was opened, putting it below our 500k threshold, it's [GH stars](https://github.com/picocss/pico) still put it within our scope. However, development seems to have stalled with [no commits to their primary repo](https://github.com/picocss/pico/graphs/commit-activity) since April.
> 
> Should we label this as [ out of scope](https://github.com/simple-icons/simple-icons/labels/out%20of%20scope) Issues or pull requests that are beyond the scope of the project for now, pending further monitoring?
> 
> This brings to light a possible issue with us using GH stars as a metric of popularity as they are not a reflection of current, real time popularity in the same way Similarweb is, rather they reflect a cumulative popularity over the lifetime of a project. Perhaps we need to revisit _how_ we use stars as a metric of popularity but that discussion should be had separately.

I agree with the rest of the reasoning, but how do we decide what the recent commit should be? Last commit being 6 months ago doesn't seem that bad to me. Maybe the maintainer took a break. Website is up and there doesn't seem to be any message about being discontinued.

**Comment by adamrusted** [comment]
Let's use the 6mth threshold for GH updates for now then. Given the main package hasn't been updated since 22nd April, that would therefore put this out of scope for the time being. Thanks regardless for your work @ripzerotwo - and if the repository receives updates in the near future, please feel free to re-open this PR!

**Pull request closed without merging**